governance/tools-transfer.md

```diff
@@ -0,0 +1,54 @@
+# Tools Transfer Guide
+
+## Outline
+This page describes the services that the tomesh tools team is responsible for.
+The acting team may use this document as a reference whenever there is a transfer of responibilities to a new group.
+Contact the tools team for further details.
+
+## Tools Team Responsibilities
+The tomesh tools team is responsible for managing the following:
+
+### Website
+- Server: All the files are hosted on the tomesh1 [Digital Ocean](https://www.digitalocean.com) droplet.
+- DNS: [hover](www.hover.com) is the DNS service provider. This service allows us to map tomesh.net to the IP address of the tomesh1 droplet.
+- Github: The [tomesh.net repository](https://github.com/tomeshnet/tomesh.net) hosts the website files.
```

Comment:
@garrying @darkdrgn2k can we add details about how the new map (files, node list, API keys) are managed?

Reply:
Information added, please modify as you see fit.